assets/js/atomic/blocks/product-elements/title/attributes.ts

```diff
@@ -12,6 +12,14 @@ let blockAttributes: Record< string, Record< string, unknown > > = {
 		type: 'boolean',
 		default: true,
 	},
+	linkTarget: {
+		type: 'string',
+		default: '_self',
```

Comment:
We shouldn't have to provide this as the default - this is a browser default and we should let browsers control that.